src/view/printer/index.ts

```diff
@@ -1,19 +1,22 @@
 /**
  * {@link https://github.com/chalk/chalk}
  */
-import * as chalk from 'chalk';
+const chalk = require('chalk');
+
 /**
  * {@link https://www.npmjs.com/package/figlet}
  */
-import * as figlet from 'figlet';
+const figlet = require('figlet');
+
 /**
  * {@link https://www.npmjs.com/package/node-emoji}
  */
-import { emoji } from 'node-emoji';
+const { emoji } = require('node-emoji');
+
 /**
  * {@link https://github.com/oclif/cli-ux}
  */
-import { cli as cliUX } from 'cli-ux';
+const cliUX = require('cli-ux').cli;
```

Comment:
require() is legacy way to import something using CommonJS. Use the ESM way for future proofing: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c